### ISSUE TITLE
Command line utility to execute Resonite bot account commands

### ISSUE THREAD

**Issue by Veeridian** [Yellow-Dog-Man/Resonite-Issues#2892]
### Is your feature request related to a problem? Please describe.

It can be tedious at times to execute a high number of commands from the Resonite bot in Resonite, and requires the user to launch the entirety of Resonite possibly just for use of these commands.

### Describe the solution you'd like

A command line utility that can be used to execute Resonite bot account commands.

### Describe alternatives you've considered

None.

### Requesters

Veer, Dante, Git

**Comment by stiefeljackal** [comment]
I know @Geenz created #1491 that is related to the mentioned request. It might be a possible duplicate.

**Comment by shiftyscales** [comment]
Closing in favour of the existing / duplicate issue #1491.

**Comment by shiftyscales** [comment]
Discussed it internally and this issue has enough differences to stand on its own, @stiefeljackal. Effectively, this is just a very simple tool to allow users to send commands to the Resonite account without needing to launch the full Resonite account.

Geenz's issue linked above has a lot more depth to it, in terms of allowing scripting of commands, etc. and would be a much larger issue.

Re-opening this one to stand as its own separate tool / work item.

**Comment by Frooxius** [comment]
Yeah we'll keep it separate for now.

Geenz's issue is more broad, with allowing script-ability, PowerShell use and all that kind of stuff.

This one will be just quick implementation (1-2 hours at most) that will help in the meanwhile, but won't be a good basis to turn it into #1491 Once that one goes through, we could potentially sunset this one, but we'll see how it goes.

**Comment by stiefeljackal** [comment]
> Discussed it internally and this issue has enough differences to stand on its own, @stiefeljackal. Effectively, this is just a very simple tool to allow users to send commands to the Resonite account without needing to launch the full Resonite account.

@shiftyscales Yep. Thank you for providing additional clarity. I was just pointing out the relationship between the two. I only mentioned possible duplicate since this request will be a quick item to setup to help satisfy a business need quickly compared to developing a full suite of commands, which you have mentioned.

> This one will be just quick implementation (1-2 hours at most) that will help in the meanwhile, but won't be a good basis to turn it into https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1491

@Frooxius Agreed.
